RingPublishingTracking/publish.gradle

```diff
@@ -1,6 +1,6 @@
-def LIB_GROUP_ID = 'com.ringpublishing.tracking'
-def LIB_ARTIFACT_ID = 'lib'
-def LIB_VERSION = "$lib_version_name"
+def LIB_GROUP_ID = 'com.ringpublishing'
+def LIB_ARTIFACT_ID = 'tracking'
```

Comment:
to jak w końcu? bo w przykładzie chciałeś
implementation("com.ringpublishing.tracking:lib:1.0.+")